docs/pages/components/date-range-picker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+
+const pageFilename = 'components/date-range-picker';
+const requireDemo = require.context(
+  'docs/src/pages/components/date-range-picker',
+  false,
+  /\.(js|tsx)$/,
+);
+const requireRaw = require.context(
+  '!raw-loader!../../src/pages/components/date-range-picker',
+  false,
+  /\.(js|md|tsx)$/,
+);
+
+// Run styled-components ref logic
+// https://github.com/styled-components/styled-components/pull/2998
+requireDemo.keys().map(requireDemo);
```

Comment:
I think I just copied this from some other component, because `yarn docs:api` didn't work for typescript sources and we discussed that it will be replaced by docs ts->docs generation.

Reply:
I'm fairly certain this is unrelated to `yarn docs:api`. We should be able the remove the highlighted blocks here and in the other demos you added.

Reply:
This is required for all the pages of the documentation